lms/djangoapps/student_account/views.py

```diff
@@ -122,7 +122,10 @@ def login_and_registration_form(request, initial_mode="login"):
         'responsive': True,
         'allow_iframing': True,
         'disable_courseware_js': True,
-        'disable_footer': True,
+        'disable_footer': not get_themed_value(
+            'ENABLE_COMBINED_LOGIN_REGISTRATION_FOOTER',
+            settings.FEATURES['ENABLE_COMBINED_LOGIN_REGISTRATION_FOOTER']
```

Comment:
> Please use
> `settings.FEATURES.get('ENABLE_COMBINED_LOGIN_REGISTRATION_FOOTER', False)`
> for the default value.

I disagree. This is a frustrating anti-pattern that is cargo-culted
throughout the platform.

Djangoapps should _not_ be setting default/fallback options; that's up
to `django.conf.settings`. Settings and their default values must be
consistent via centralization.

By specifying a default value on every `get` call [1], you run the risk
of these invocations falling out of sync of one another. We've been
burnt by this in `edx-platform` before. If you default a value to
`False` when you first roll out a feature, but then later switch it to
enabled by default, you risk diverging default values.  This is further
exacerbated by the presence of both `configuration` and
`configuration-secure` which means default values can already be
specified in no less than 3 places. Specifying defaults directly in-app
just makes this worse; that's unmaintainable and ripe for
confusion/complication.

I understand the rationale that leads developers to "look before they
leap" and safely index into collections ("safety first!"), but in this
case, `KeyError`s are actually a feature, not a bug. As this code
already explicitly assigns a default value in `lms/envs/common.py`, this
key will _always_ be present. The only way this key could possibly be
unset [2] is if something has gone terribly wrong. In that event, the
call-stack blowing `KeyError` is what we want; the error shouldn't be
silently swallowed; we need to be alerted immediately because there's a
deep-seated configuration error, not a just a simple runtime error.

[1] whether `item.get(key, default)` or `getattr(item, key, default)`
[2] `del item[key]` or `item.pop(key)` or some such